doc/mlaunch.rst

```diff
@@ -99,6 +99,20 @@ Usage
                 [--hostname HOSTNAME] [--auth] [--username USERNAME]
                 [--password PASSWORD] [--auth-db DB]
                 [--auth-roles [ROLE [ROLE ...]]]
+                [--initial-user] [--sslCAFile SSLCAFILE]
+                [--sslCRLFile SSLCRLFILE] [--sslAllowInvalidHostnames]
+                [--sslAllowInvalidCertificates] [--sslOnNormalPorts]
```

Comment:
This option has been deprecated since v2.6
https://docs.mongodb.com/manual/reference/configuration-options/#net.ssl.sslOnNormalPorts
Do we need to make it available ?

Reply:
Good call, removed.